.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,55 @@
+name: Ansible Lint
+
+on: [push, pull_request]
+
+jobs:
+  build:
```

Comment:
@bedroge Can you rename this to `ansible-lint` in a follow-up PR where you fix the linting issues?

Once that is merged, I'll change the requirements for the `master` branch that the linter check passes before PRs can be merged...